testing/python/cpu/test_tilelang_cpu_arch_config.py

```
@@ -0,0 +1,273 @@
+    with (
+        pytest.raises(
+            ValueError,
+            match="Conflicting function-level tilelang_pass_configs found in IRModule",
+        ),
+        tvm.target.Target("c"),
+    ):
+        tilelang.lower(mod, target="c", target_host="c")
+
+
+def test_lower_rejects_unregistered_pass_config_key():
+    with pytest.raises(AttributeError, match="Invalid config option 'tl.not_registered'"), tvm.target.Target("c"):
```

Ruff (0.15.9)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)